### PR TITLE
Fix some small errors (see commits)

### PR DESCRIPTION
**Pull request by seb-elico**
Hi, I'm Seb from Elico Corp. Eric contacted Vincent last week. We've tested the modules and identified a few issues. Please check our patches!

### CODE REVIEW

File: contract_agent_interface/partner.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def _get_analytic_lines(self, cr, uid, ids, field_name,
             'Payments'),
         'all_bank_ids': fields.one2many(
             'res.partner.bank', 'partner_id', 'Banks',
-            domain=['|', ('active', '=', False), ('active', '=', True)],
```
Comment (veloutin) [inline contract_agent_interface/partner.py:93]: The original value is the right one here. The reason there is this domain is unless you have a test for the field "active" in your domain, inactive entries will never be returned by the ORM.

In this case we want to be able, through the all_bank_ids field, to view all bank accounts for a partner. The rest of the code can keep on using bank_ids and never see the inactive ones.

The "active" field on res.partner.bank exists in the base module.

File: contract_isp/contract.py [hunk 2]
```diff
@@ -589,10 +589,14 @@ def create(self, cr, uid, values, context=None):
...
-            values['parent_id'] = partner_obj.read(
+            aa_ids = partner_obj.read(
                 cr, uid, values['partner_id'],
                 fields=['partner_analytic_account_id'],
-                context=context)['partner_analytic_account_id'][0]
+                context=context)['partner_analytic_account_id']
+            if aa_ids and len(aa_ids) > 0:
+                values['parent_id'] = aa_ids[0]
+            else:
+                values['parent_id'] = None
```
Comment (veloutin) [inline contract_isp/contract.py]: Should this be False instead?

File: contract_agent_interface/partner.py (hunk 1 above)
Comment (seb-elico) [inline contract_agent_interface/partner.py:93]: Thanks for your explanation. I'll cancel this commit then.

However, after installing the modules from 7.0_contract_agent_interface branch, we get an error message indicating the "active" field doesn't exist for res.partner.bank, hence my merge request. I'll investigate and come back to you if I find more information.

File: contract_isp/contract.py (hunk 2 above)
Comment (seb-elico) [inline contract_isp/contract.py]: It might as well be "False", I'm very new to OE/Odoo ORM, sorry! I'll update the merge request.

**Comment by seb-elico** [comment]
Hi Vincent, I fixed the code based on your feedback. Best regards

**Comment by veloutin** [comment]
Hello Seb, let me know how your investigation goes for the "active" field.

**Pull request merged**